plugins/arElasticSearchPlugin/lib/arElasticSearchPlugin.class.php

```diff
@@ -427,12 +463,24 @@ public function delete($object)
         }
 
         if ($this->batchMode) {
+            $indexName = get_class($object);
```

Comment:
I just noticed, this needs to happen before the `if`, as it's also used in the `else`.